### DISCUSSION TITLE
How to install experiment extension in python3 pilot

### DISCUSSION THREAD

**Discussion by zhangxiaomei** [DIRACGrid/DIRAC#6107] [Q&A] [answered]
I am using python3 DIRAC7.3 server, and try to test with python3 pilot.
With the configuration:
Operations/setup/pilot/Extensions=IHEPDIRAC
Operations/setup/pilot/Version= IHEPDIRAC version
It only works with DIRAC  version, not IHEPDIRAC version.
How can I make IHEPDIRAC installed by python3 pilot? Thanks.

Regards,
Xiaomei

**Comment by fstagni** [comment, marked as answer]
Try with
```
Operations/<setup>/Pilot/Project=IHEP
```
and let me know.

**Reply by zhangxiaomei** [reply]
It works. Thanks.
